Android/app/src/main/java/adapters/ContributorsAdapter.java

```diff
@@ -0,0 +1,73 @@
+package adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.github.project_travel_mate.R;
+import objects.Contributor;
+
+/**
+ * @author amoraitis
+ */
+public class ContributorsAdapter extends BaseAdapter {
+    private final Context mContext;
+    private List<Contributor> mContributors;
+
+    public ContributorsAdapter(Context context, List<Contributor> contributors) {
+        mContext = context;
+        mContributors = contributors;
+    }
+
+    @Override
+    public int getCount() {
+        return mContributors.size();
+    }
+
+    @Override
+    public Object getItem(int i) {
+        return mContributors.get(i);
+    }
+
+    @Override
+    public long getItemId(int i) {
+        return 0;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        final Contributor contributor = mContributors.get(position);
+        if (convertView == null) {
+            final LayoutInflater layoutInflater = LayoutInflater.from(mContext);
+            convertView = layoutInflater.inflate(R.layout.contributors_gridview_item, null);
+        }
+
+        final ImageView contributor_avatarImageView = (ImageView) convertView.findViewById(R.id.contributor_image);
```

Comment:
Use butterknife

Reply:
Does this include to implement a viewholder class, too?

Reply:
I don't think it will. I have used butterknife with fragments in this project, and there were no issues.

Reply:
> I don't think it will. I have used butterknife with fragments in this project, and there were no issues.

I trust u, but my Logcat doesn't agree...